harvester/edurep/extraction.py

```diff
@@ -229,3 +229,12 @@ def get_disciplines(cls, soup, el):
     def get_humanized_disciplines(cls, soup, el):
         blocks = cls.find_all_classification_blocks(el, "discipline", "czp:entry")
         return list(set([block.find('czp:langstring').text.strip() for block in blocks]))
+
+    @classmethod
+    def get_analysis_allowed(cls, soup, el):
+        node = el.find('czp:copyrightandotherrestrictions')
+        value = node.find('czp:value').find('czp:langstring').text.strip() if node else None
+        if value:
```

Comment:
Volgens mij moet de default False zijn, want als je de rechten niet weg geeft, dan heb je onder de Nederlandse wet auteursrechten en mogen we niet zomaar handelen.

Een klein dingetje is dat je het iets bondiger kan opschrijven. Dat vind ik zelf makkelijker lezen. Geen blocker.
`return value and "-nd" not in value`